samples/keypublisher/README.md

```diff
@@ -0,0 +1,291 @@
+# User-managed encryption key publisher
+
+Code intended to be run by users of the Live Stream API. This code utilizes DRM
+provider CPIX APIs to fetch encryption keys and store them in Google Secret
+Manager for use with the Live Stream API's encryption feature.
+
+## Setup
+
+### Install and configure gcloud
+
+Follow standard install instructions. Use `gcloud config` to set your project
+and preferred region/zone.
+
+### Configure Cloud Functions and API Gateway
+
+### Enable APIs
+
+Enable Google APIs used to host your key publisher and write key information to
+Secret Manager.
+
+```shell
+gcloud services enable apigateway.googleapis.com
+gcloud services enable cloudbuild.googleapis.com
+gcloud services enable cloudfunctions.googleapis.com
+gcloud services enable secretmanager.googleapis.com
+gcloud services enable servicecontrol.googleapis.com
+gcloud services enable servicemanagement.googleapis.com
+```
+
+### Create service accounts
+
+Two service accounts are needed for your key publisher. The first allows your
+API Gateway to invoke your cloud function, and the second allows Cloud Functions
+to read and write to Secret Manager.
+
+Create the service account for invoking Cloud Functions:
+
+```shell
+gcloud iam service-accounts create livestream-cf-invoker \
+  --display-name="Cloud Function invoker for the API Gateway" \
+  --description="Service Account to be used by the API Gateway to invoke Cloud Functions"
+```
+
+Then configure the IAM policy binding:
+
+```shell
+gcloud projects add-iam-policy-binding my-project \
```

Comment:
I suggest replacing every occurance of `my-project` with `[MY-PROJECT]`. This way it stands out much more and is more obvious that it should be replaced by the user.

Reply:
agreed, thank you.